### PR TITLE
Client Fetches data from the server

### PR DESCRIPTION
**Pull request by dyliman**
@Happypals @imtryingguys @serviolee Please review this for me. Let me know if anything looks off. Client is able to fetch data from the server but right now client is not using the data

### CODE REVIEW

**Review by ngudbranson — APPROVED** [review]
Only question I have is what's cors and how does it influence your data?

**Comment by dyliman** [comment]
@imtryingguys cors is short for Cross-origin resource sharing and it just fixes the problem I have when my client makes a request to a server and it says something like Cross-Origin Read Blocking. Is there a better way to fix that issue?

**Review by serviolee — APPROVED** [review]
looks good! minor thing, not sure if a callback is needed for the insertInfo since nothing is being returned

File: database/index.js [hunk 1]
```diff
@@ -28,4 +35,5 @@ const insertInfo = (callback, insert) => {
 
 module.exports = {
   insertInfo,
+  getReviewInfo,
```
Comment (Happypals) [inline database/index.js:38]: the comma behind getReviewInfo does not break the code?

File: server/index.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const db = require('../database/index.js');
+var cors = require('cors')
```
Comment (Happypals) [inline server/index.js:4]: be consistent with semicolon here?

File: server/index.js [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
 
 const app = express();
 
 app.use(express.static('client'));
+app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({
   extended: true,
```
Comment (Happypals) [inline server/index.js:12]: Normally, I would not add a comma if it is the last entry in an object

File: client/index.jsx [hunk 4]
```diff
@@ -1 +1,36 @@
...
+import ReactDOM from 'react-dom';
+// import $ from 'jquery';
 
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentGame: 1,
+      reviews: [],
```
Comment (Happypals) [inline client/index.jsx:11]: same here, the comma

**Comment by Happypals** [comment]
If you use app.use( express.static()). The cors is not necessary because
both your client and server will share the same origin. But probably they
don't want us to use express to host our client. Or you can add header,
refer to this website: https://enable-cors.org/server_expressjs.html. That's
what I used to make it work. But if it is working, it does not bother me at
all.

**Comment by Happypals** [comment]
I see you added the line app.use(express.static('client'));
I guess cors is probably not necessary if you are serving the client using
express already

**Pull request merged**